src/python/pants/notes/2.9.x.md

```diff
@@ -1,5 +1,45 @@
 # 2.9.x Release Series
 
+## 2.9.0rc0 (Dec 17, 2021)
+
+### New Features
+
+* Add `--docker-run-args` option to support `docker run [args] <image>`. ([#13818](https://github.com/pantsbuild/pants/pull/13818))
+
+* An include_requirements field on pex_binary. ([#13894](https://github.com/pantsbuild/pants/pull/13894))
+
+* Add new `secrets` field to `docker_image`. ([#13830](https://github.com/pantsbuild/pants/pull/13830))
+
+* [jvm] Add support for Scalatest ([#13872](https://github.com/pantsbuild/pants/pull/13872))
+
+### User API Changes
+
+* Upgrade Pex to 2.1.58. ([#13915](https://github.com/pantsbuild/pants/pull/13915))
+
+### Bug fixes
+
+* [jvm] Apply the resolve for `deploy_jar` and restore validation of resolve compatibility ([#13911](https://github.com/pantsbuild/pants/pull/13911))
+
+* Fix rendering of default field values in reference docs. ([#13891](https://github.com/pantsbuild/pants/pull/13891))
```

Comment:
Nit: documentation

Reply:
Ok. It's clearly both, so I don;t get the algorithm, but I'll make it so.